src/MorganStanley.ComposeUI.MainApp/MorganStanley.ComposeUI.MainApp.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<!--
```

Comment:
Should we call this something like AvaloniaApp instead of just app to distinguish this?  Will we have a WPF app project?

Reply:
We can call it AvaloniaShell